aiodns/__init__.py

```diff
@@ -204,7 +204,7 @@ def query(
 
     def query(
         self, host: str, qtype: str, qclass: Optional[str] = None
-    ) -> asyncio.Future[list[Any]]:
+    ) -> asyncio.Future[list[Any] | Any]:
```

Comment:
I think we need this to work correctly with mypy's type checking.

```suggestion
    ) -> asyncio.Future[list[Any]] | asyncio.Future[Any]:
```